### PR TITLE
Mapmodify 4

### LINKED ISSUE #975: Turret Autofill Toggle in config.lua has no effect.
Well this 
`
-- autofill turrets with ammo    
    autofill = {
        enabled = false
    },
`
does nothing. Case and point the Ribbons and Recursion run on s8.

To recreate simply make new scenario with autofill off and give yourself some turrets via the editor.

### LINKED ISSUE #984: Map Previews: rectangular_spiral.png should be renamed square_spiral.png
New minor one for me to fix, but am not at home so adding an issue to remind me to do it.

Investigate names of square_spiral.lua in map files compared to rectangular_spiral.png in map_gen/data/.map_previews. It caused some confusion in #helpdesk-and-moderation today in relation to the popular Youtube playthrough by Vueltero.

image

### PR DESCRIPTION
**Pull request by theorangeangle**
The fourth edition of "Orange tries to fix all our broken maps and add descriptions".

- [x] fixed broken image sprite and replaced with a parity for _diggy_

- [x] disabled landfill on _connected dots_, except light water tiles

- [x]  decreased the 83x pollution increase on _danger ores_

- [x]  added disclaimer on the _default_ map, printed in chat

- [x] Map description and info for _fish islands_

- [x] Map description and info for _fractal balls_

- [x] Map description and info for _loading screen_ (previously "factory")

- [x] Map preview for _loading screen_ (previously "factory")
- [x] Map description and info for _fruit loops_

- [x] Map description and info for _gears_
- [ ] tweaked biter balance on _double beach_ (less early game, more late game)

And a whole bunch of other maps, but I want to get these changes through sooner rather than later if possible. 

closes #975
closes #984

### CODE REVIEW

**Comment by theorangeangle** [comment]
This will be reopened as MM #5 soon 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SimonFlapse** [comment]
~~This pull request have been pretty inflated, it's a bit hard to keep track of if the changes are correct.~~
I've fixed it.

File: map_gen/maps/diggy/feature/diggy_cave_collapse.lua [hunk 1]
```diff
@@ -335,8 +335,9 @@ end
 
     @param global_config Table {@see Diggy.Config}.
 ]]
+--Special thanks to justarandomgeek from the main factorio discord guild for helping to teach orange how image classes work.
```
Comment (SimonFlapse) [inline map_gen/maps/diggy/feature/diggy_cave_collapse.lua:338]: You can use the /rich-text command to access the rich-text gui which can generate the tags for you, just copy the image and it'll place the string in your clipboard

image

**Review by SimonFlapse — APPROVED** [review]
I'm good with the current state of the PR, you can always include the missing things in a new PR
@theorangeangle

**Comment by theorangeangle** [comment]
@SimonFlapse i'm good to merge but can you check the commits from july 17? I want to make sure nothing becomes weird because the commit structure seems a little off to me

**Comment by SimonFlapse** [comment]
> @SimonFlapse i'm good to merge but can you check the commits from july 17? I want to make sure nothing becomes weird because the commit structure seems a little off to me

Everything is okay, it's just because you pulled some commits from other branches than the develop branch. I've looked through the files affected by this PR and everything is alright

**Pull request merged**